doc/src/manual/command-line-interface.md

```diff
@@ -195,7 +195,7 @@ The following is a complete list of command-line switches available when launchi
 |`-g`, `--debug-info={0,1*,2}`          |Set the level of debug info generation (level is 2 if `-g` is used without a level) ($)|
 |`--inline={yes\|no}`                   |Control whether inlining is permitted, including overriding `@inline` declarations|
 |`--check-bounds={yes\|no\|auto*}`      |Emit bounds checks always, never, or respect `@inbounds` declarations ($)|
-|`--math-mode={ieee,fast}`              |Disallow or enable unsafe floating point optimizations (overrides `@fastmath` declaration)|
+|`--math-mode={ieee,user}`              |Disallow or allow unsafe floating point optimizations (overrides `@fastmath` declaration)|
```

Comment:
Maybe something like this would be slightly more clear?

```suggestion
|`--math-mode={ieee,user}`              |Always follow `ieee` floating point semantics or respect `@fastmath` declarations|
```